datadog_checks_dev/datadog_checks/dev/tooling/release.py

```diff
@@ -118,19 +118,28 @@ def update_agent_requirements(req_file, check, newline):
 
 def build_package(package_path, sdist):
     with chdir(package_path):
-        # Clean up: Files built previously and now deleted might still persist in build directory
-        # and will be included in the final wheel. Cleaning up before avoids that.
-        result = run_command([sys.executable, 'setup.py', 'clean', '--all'], capture='out')
-        if result.code != 0:
-            return result
-
-        result = run_command([sys.executable, 'setup.py', 'bdist_wheel', '--universal'], capture='out')
-        if result.code != 0:
-            return result
-
-        if sdist:
-            result = run_command([sys.executable, 'setup.py', 'sdist'], capture='out')
+        if file_exists(path_join(package_path, 'pyproject.toml')):
```

Comment:
What do you think about making this a utility function?

Reply:
Hmm, AFAICT checking the file given a directory only happens here. Is that not the case?

Reply:
It does, but in some other places there's a utility function that's only used once. But those are more complex like `normalize_project_name` so it's probably not needed.